suse_migration_services/units/prepare.py

```diff
@@ -251,3 +262,23 @@ def get_root_disk_device(root_path):
                 block_type = block_record[1]
                 if block_type in considered_block_types:
                     return block_record[0]
+
+
+def get_regionsrv_client_file_location(root_path):
+    """
+    Find the correct path for the cloud-regionsrv-client cache files
+    as the location moves from /var/lib/cloudregister
+    (pre cloud-regionsrv-client-10.0.4) to /var/cache/cloudregister
+    (post cloud-regionsrv-client-10.0.4)
+    """
+    for cloud_register_path in [
+        os.sep.join([root_path, 'var', 'cache', 'cloudregister']),
+        os.sep.join([root_path, 'var', 'lib', 'cloudregister'])
+    ]:
+        if os.path.isdir(cloud_register_path) and \
+                any('SMT' in x for x in os.listdir(cloud_register_path)):
+            return cloud_register_path
+    raise DistMigrationZypperMetaDataException(
+        'No cloud-regionsrv-client cache files found in '
+        '/var/cache/cloudregister or /var/cache/cloudregister'
```

Comment:
One of the paths in the exception should be `/var/lib/cloudregister` (I guess copypasta)

Reply:
yup, copy paste error, thanks for catching